Bottom: 0c2299a3bc0d59f23b2b1ceb63cf4e3ab6f2d1aa
Top:    11658bcb33cb5a92eb73d56aa2f0dd812c758e9b
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-21 19:05:24 +0000

Refresh of re-enable-soldermask-emission

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index efe337c..8d81a8e 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1040,7 +1040,7 @@ object3d_from_soldermask_within_area (POLYAREA *area, int side)
   LayerType *layer;
   POLYAREA *pa;
 
-  return NULL;
+//  return NULL;
 
   poly_Copy0 (&info.poly, area);
   info.side = side;
